exercises/bank-account/src/main/scala/Bank.scala

```diff
@@ -0,0 +1,13 @@
+trait BankAccount {
+
+  def closeAccount(): Unit
+
+  def getBalance: Option[Int]
+
+  def incrementBalance(increment: Int): Option[Int]
+}
+
+object Bank {
```

Comment:
Well, there is a `Bank` object being implemented of course, so that sort of makes sense. I usually named the files after the slug name, so for the `BankAccount` exercise, I would use `BankAccount.scala`.